modules/storage-persistent-storage-pv.adoc

```diff
@@ -128,16 +128,17 @@ the Pods that use these volumes are deleted.
 |AWS EBS  | ✅ | - |  -
 |Azure File | ✅ | ✅ | ✅
 |Azure Disk | ✅ | - | -
```

Comment:
Azure (both file and disk) are not supported in OCP 4.1

Reply:
Both of these lines have been commented out.